readme.md

```diff
@@ -68,7 +67,13 @@ Instructors will be checking Github issues to respond to technical questions.
 
 Additionally, each group may request 20 minutes of in-person instructor support **3 times** over the course of the project. These may be redeemed **anytime** during business hours: simply Slack the instructor and they'll come as quickly as possible. However, unused minutes don't "roll over".
 
-For existential questions, and/or those regarding expectations, instructors may be reached via Slack at **any time**, inside **and** outside of business hours (although they may be somewhat slower to respond outside of business hours).
+???mms  ^ no mention of tokens, but does mention "redeeming".
+Did you remove because it wasn't clear or because we aren't using them?
+Below is the original, with "each is good for" added - for clarity.
+
+Additionally, each group will receive 3 "tokens", each is good for 20 minutes of in-person instructor support. These may be redeemed any time during business hours: simply Slack the instructor and they'll come as quickly as possible. Unused minutes don't "roll over".
+
+For existential questions, and/or those regarding expectations, instructors may be reached via Slack at **any time**, inside **and** outside of business hours (although they may be somewhat slower to respond outside of business hours).  We recommend contacting the group of instructors via ???.
```

Comment:
Maybe say they should contact me first, and if I don't respond within 10 minutes, to contact Jesse?